File: assets/rest/samples/python/DelegatesAPI.getVoters.get.py
from risesdk import PublicKey, APIError, Client

api = Client('https://wallet.rise.vision/api/')
delegate_pk = PublicKey.fromhex('a65a8160b1e0733f66d1f1a8f322c9af29b26d5e491a84d6e3ae0ec43e000446')
try:
    voters = api.delegates.get_voters(delegate_pk)
    print(len(voters)) # 5
except APIError as err:
    print('API error:', err)

File: assets/rest/samples/elixir/DelegatesAPI.getVoters.get.exs
import RiseElixir

try do
  result =
    RiseElixir.Delegates.get_voters(%{
      publicKey: "7f31f1b13f211ada60b61e47778b8d59e7f0ecb56fd6f9ea9200c805e352f6a8"
    })

  IO.puts(Enum.count(result["accounts"]))
catch
  err -> "API error #{err}"
end
